src/components/Rankings.js

```diff
@@ -19,10 +21,37 @@ function Rankings({ coverageData }) {
     }
   }
 
+  const SORT_FUNCTIONS = {
+    Ascending: (a, b) => a.percentage - b.percentage,
+    Descending: (a, b) => b.percentage - a.percentage,
+    Alphabetical: (a, b) => a.name.localeCompare(b.name),
+    ReverseAlphabetical: (a, b) => b.name.localeCompare(a.name),
+  };
+
+  function changeSort(event) {
+    setFields([...fields].sort(SORT_FUNCTIONS[event.target.value]));
+  }
+
   return (
     <div className="flex-auto">
       <div className="bg-white p-2 rounded-widgit">
-        <h1 className="font-bold text-xl pb-3">Rankings</h1>
+        <div className="flex flex-row justify-between">
+          <h1 className="font-bold text-xl pb-3">Rankings</h1>
+          <select id="sortSelect" onChange={changeSort}>
```

Comment:
Let's add some styling to the container so it looks less jarring. A few things to consider would be: 
- A bg color that matches the container 
- A border to visually separate the sort logic
- A border color that's not too dark, maybe one of the grays that Alden mentions in his design docs
- A border-radius to match the rounded aesthetic of the rest of the page
- X-margin so the sort never gets too close to the Ranking heading 
- X-padding so that the selected option doesn't get too close to the border

If you think of anything else based on Alden's designs, feel free to add that as well.